bin/cron/sync_dropbox

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+# This syncs content changes from the shared dropbox folder to our github repo
+# on the staging machine. This is required for contents changes from dropbox to
+# be deployed.
+
+system("sudo apt-get install -y unison")
+
+FOLDERS = {
+  "advocacy.code.org" => "/home/ubuntu/staging/pegasus/sites.v3/advocacy.code.org",
```

Comment:
Might be useful to use a constant for the project root (`/home/ubuntu/staging`)

Reply:
Should it be `/home/ubuntu/Dropbox/Shared`?